<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="19" ignored="375" total="395" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Create with ref Global Only-SAP Direct Copy with Rejections" duration-ms="41844" started-at="2019-03-11T07:03:35Z" finished-at="2019-03-11T07:04:17Z">
    <groups>
    </groups>
    <test name="HeiPort_Login_LDR" duration-ms="5746" started-at="2019-03-11T07:03:35Z" finished-at="2019-03-11T07:03:40Z">
      <class name="com.mendix.test.LoginScript">
        <test-method status="PASS" signature="tasKill()[pri:0, instance:com.mendix.test.LoginScript@62ee68d8]" name="tasKill" duration-ms="66" started-at="2019-03-11T12:33:35Z" finished-at="2019-03-11T12:33:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tasKill -->
        <test-method status="FAIL" signature="openBrowser()[pri:0, instance:com.mendix.test.LoginScript@62ee68d8]" name="openBrowser" duration-ms="4877" started-at="2019-03-11T12:33:35Z" finished-at="2019-03-11T12:33:40Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens

	at com.mendix.page.VendorPage_NAV.<init>(VendorPage_NAV.java:1351)
	at com.mendix.tool.PageContainer.initPages(PageContainer.java:49)
	at com.mendix.tool.PageContainer.<init>(PageContainer.java:36)
	at com.mendix.tool.SharedDriver.createDriver(SharedDriver.java:27)
	at com.mendix.test.LoginScript.openBrowser(LoginScript.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="loginAsLDR(java.util.Map)[pri:0, instance:com.mendix.test.LoginScript@62ee68d8]" name="loginAsLDR" duration-ms="15" started-at="2019-03-11T12:33:40Z" data-provider="HeiPort_Login" finished-at="2019-03-11T12:33:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.LoginScript.loginAsLDR(LoginScript.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsLDR -->
      </class> <!-- com.mendix.test.LoginScript -->
    </test> <!-- HeiPort_Login_LDR -->
    <test name="Create_with_ref_Global_Only_SAP_Direct_Copy_with_Rejections" duration-ms="117" started-at="2019-03-11T07:03:41Z" finished-at="2019-03-11T07:03:41Z">
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="Material_Direct_Copy_With_References(java.util.Map)[pri:0, instance:com.mendix.test.MaterialScript@cb5822]" name="Material_Direct_Copy_With_References" duration-ms="2" started-at="2019-03-11T12:33:41Z" data-provider="Process_Information_Check" finished-at="2019-03-11T12:33:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.Material_Direct_Copy_With_References(MaterialScript.java:735)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Material_Direct_Copy_With_References -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Create_with_ref_Global_Only_SAP_Direct_Copy_with_Rejections -->
    <test name="Process_Information_Check" duration-ms="5586" started-at="2019-03-11T07:03:41Z" finished-at="2019-03-11T07:03:46Z">
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="Process_Information_Check(java.util.Map)[pri:0, instance:com.mendix.test.MaterialScript@224edc67]" name="Process_Information_Check" duration-ms="5504" started-at="2019-03-11T12:33:41Z" data-provider="Process_Information_Check" finished-at="2019-03-11T12:33:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.Process_Information_Check(MaterialScript.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Process_Information_Check -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Process_Information_Check -->
    <test name="HeiPort_Login_GDA" duration-ms="3127" started-at="2019-03-11T07:03:46Z" finished-at="2019-03-11T07:03:49Z">
      <class name="com.mendix.test.LoginScript">
        <test-method status="FAIL" signature="openBrowser()[pri:0, instance:com.mendix.test.LoginScript@523884b2]" name="openBrowser" duration-ms="3113" started-at="2019-03-11T12:33:46Z" finished-at="2019-03-11T12:33:49Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens

	at com.mendix.page.VendorPage_NAV.<init>(VendorPage_NAV.java:1351)
	at com.mendix.tool.PageContainer.initPages(PageContainer.java:49)
	at com.mendix.tool.PageContainer.<init>(PageContainer.java:36)
	at com.mendix.tool.SharedDriver.createDriver(SharedDriver.java:27)
	at com.mendix.test.LoginScript.openBrowser(LoginScript.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="loginAsGDA()[pri:0, instance:com.mendix.test.LoginScript@523884b2]" name="loginAsGDA" duration-ms="7" started-at="2019-03-11T12:33:49Z" finished-at="2019-03-11T12:33:49Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.LoginScript.loginAsGDA(LoginScript.java:195)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsGDA -->
      </class> <!-- com.mendix.test.LoginScript -->
    </test> <!-- HeiPort_Login_GDA -->
    <test name="Reject_Material_Request" duration-ms="81" started-at="2019-03-11T07:03:49Z" finished-at="2019-03-11T07:03:50Z">
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="Material_Reject_GDA(java.util.Map)[pri:0, instance:com.mendix.test.MaterialScript@3532ec19]" name="Material_Reject_GDA" duration-ms="1" started-at="2019-03-11T12:33:50Z" data-provider="Process_Information_Check" finished-at="2019-03-11T12:33:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.Material_Reject_GDA(MaterialScript.java:708)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Material_Reject_GDA -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Reject_Material_Request -->
    <test name="Process_Information_Check_After_Reject" duration-ms="5584" started-at="2019-03-11T07:03:50Z" finished-at="2019-03-11T07:03:55Z">
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="Process_Information_Check(java.util.Map)[pri:0, instance:com.mendix.test.MaterialScript@3e6fa38a]" name="Process_Information_Check" duration-ms="5501" started-at="2019-03-11T12:33:50Z" data-provider="Process_Information_Check" finished-at="2019-03-11T12:33:55Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.Process_Information_Check(MaterialScript.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Process_Information_Check -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Process_Information_Check_After_Reject -->
    <test name="HeiPort_ReLogin_LDR" duration-ms="3281" started-at="2019-03-11T07:03:55Z" finished-at="2019-03-11T07:03:58Z">
      <class name="com.mendix.test.LoginScript">
        <test-method status="FAIL" signature="openBrowser()[pri:0, instance:com.mendix.test.LoginScript@5e3a8624]" name="openBrowser" duration-ms="3208" started-at="2019-03-11T12:33:55Z" finished-at="2019-03-11T12:33:58Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens

	at com.mendix.page.VendorPage_NAV.<init>(VendorPage_NAV.java:1351)
	at com.mendix.tool.PageContainer.initPages(PageContainer.java:49)
	at com.mendix.tool.PageContainer.<init>(PageContainer.java:36)
	at com.mendix.tool.SharedDriver.createDriver(SharedDriver.java:27)
	at com.mendix.test.LoginScript.openBrowser(LoginScript.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="loginAsLDR(java.util.Map)[pri:0, instance:com.mendix.test.LoginScript@5e3a8624]" name="loginAsLDR" duration-ms="8" started-at="2019-03-11T12:33:58Z" data-provider="HeiPort_Login" finished-at="2019-03-11T12:33:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.LoginScript.loginAsLDR(LoginScript.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsLDR -->
      </class> <!-- com.mendix.test.LoginScript -->
    </test> <!-- HeiPort_ReLogin_LDR -->
    <test name="Material_Resubmit_To_Approval" duration-ms="4" started-at="2019-03-11T07:03:59Z" finished-at="2019-03-11T07:03:59Z">
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="navigate_To_MDM_Workflow()[pri:0, instance:com.mendix.test.MaterialScript@7cd62f43]" name="navigate_To_MDM_Workflow" duration-ms="1" started-at="2019-03-11T12:33:59Z" finished-at="2019-03-11T12:33:59Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.navigate_To_MDM_Workflow(MaterialScript.java:673)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigate_To_MDM_Workflow -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Material_Resubmit_To_Approval -->
    <test name="Process_Information_ReCheck_After_Resubmit" duration-ms="5569" started-at="2019-03-11T07:03:59Z" finished-at="2019-03-11T07:04:04Z">
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="Process_Information_Check(java.util.Map)[pri:0, instance:com.mendix.test.MaterialScript@1794d431]" name="Process_Information_Check" duration-ms="5502" started-at="2019-03-11T12:33:59Z" data-provider="Process_Information_Check" finished-at="2019-03-11T12:34:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.Process_Information_Check(MaterialScript.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Process_Information_Check -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Process_Information_ReCheck_After_Resubmit -->
    <test name="HeiPort_ReLogin_GDA" duration-ms="3331" started-at="2019-03-11T07:04:04Z" finished-at="2019-03-11T07:04:08Z">
      <class name="com.mendix.test.LoginScript">
        <test-method status="FAIL" signature="openBrowser()[pri:0, instance:com.mendix.test.LoginScript@1a968a59]" name="openBrowser" duration-ms="3305" started-at="2019-03-11T12:34:04Z" finished-at="2019-03-11T12:34:07Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens

	at com.mendix.page.VendorPage_NAV.<init>(VendorPage_NAV.java:1351)
	at com.mendix.tool.PageContainer.initPages(PageContainer.java:49)
	at com.mendix.tool.PageContainer.<init>(PageContainer.java:36)
	at com.mendix.tool.SharedDriver.createDriver(SharedDriver.java:27)
	at com.mendix.test.LoginScript.openBrowser(LoginScript.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="loginAsGDA()[pri:0, instance:com.mendix.test.LoginScript@1a968a59]" name="loginAsGDA" duration-ms="21" started-at="2019-03-11T12:34:07Z" finished-at="2019-03-11T12:34:08Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.LoginScript.loginAsGDA(LoginScript.java:195)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsGDA -->
      </class> <!-- com.mendix.test.LoginScript -->
    </test> <!-- HeiPort_ReLogin_GDA -->
    <test name="Material_Create_Review_Global_Data_Approve_GDA" duration-ms="78" started-at="2019-03-11T07:04:08Z" finished-at="2019-03-11T07:04:08Z">
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="Material_Create_Review_Global_Data_Approve_GDA(java.util.Map)[pri:0, instance:com.mendix.test.MaterialScript@4dfa3a9d]" name="Material_Create_Review_Global_Data_Approve_GDA" duration-ms="2" started-at="2019-03-11T12:34:08Z" data-provider="Process_Information_Check" finished-at="2019-03-11T12:34:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.Material_Create_Review_Global_Data_Approve_GDA(MaterialScript.java:205)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Material_Create_Review_Global_Data_Approve_GDA -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Material_Create_Review_Global_Data_Approve_GDA -->
    <test name="Process_Information_Check_Syndication" duration-ms="5564" started-at="2019-03-11T07:04:08Z" finished-at="2019-03-11T07:04:13Z">
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="Process_Information_Check(java.util.Map)[pri:0, instance:com.mendix.test.MaterialScript@57fffcd7]" name="Process_Information_Check" duration-ms="5505" started-at="2019-03-11T12:34:08Z" data-provider="Process_Information_Check" finished-at="2019-03-11T12:34:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.Process_Information_Check(MaterialScript.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Process_Information_Check -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Process_Information_Check_Syndication -->
    <test name="HeiPort_Login_LDR_Syndication" duration-ms="3271" started-at="2019-03-11T07:04:13Z" finished-at="2019-03-11T07:04:17Z">
      <class name="com.mendix.test.LoginScript">
        <test-method status="FAIL" signature="openBrowser()[pri:0, instance:com.mendix.test.LoginScript@4f7d0008]" name="openBrowser" duration-ms="3205" started-at="2019-03-11T12:34:13Z" finished-at="2019-03-11T12:34:16Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	Syntax error on tokens, delete these tokens

	at com.mendix.page.VendorPage_NAV.<init>(VendorPage_NAV.java:1351)
	at com.mendix.tool.PageContainer.initPages(PageContainer.java:49)
	at com.mendix.tool.PageContainer.<init>(PageContainer.java:36)
	at com.mendix.tool.SharedDriver.createDriver(SharedDriver.java:27)
	at com.mendix.test.LoginScript.openBrowser(LoginScript.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="loginAsLDR(java.util.Map)[pri:0, instance:com.mendix.test.LoginScript@4f7d0008]" name="loginAsLDR" duration-ms="8" started-at="2019-03-11T12:34:16Z" data-provider="HeiPort_Login" finished-at="2019-03-11T12:34:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.LoginScript.loginAsLDR(LoginScript.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsLDR -->
      </class> <!-- com.mendix.test.LoginScript -->
    </test> <!-- HeiPort_Login_LDR_Syndication -->
    <test name="Material_Create_Syndication_Check" duration-ms="51" started-at="2019-03-11T07:04:17Z" finished-at="2019-03-11T07:04:17Z">
      <class name="com.mendix.test.LoginScript">
        <test-method status="PASS" signature="setup(org.testng.ITestContext)[pri:0, instance:com.mendix.test.LoginScript@4f7d0008]" name="setup" is-config="true" duration-ms="116" started-at="2019-03-11T12:33:35Z" finished-at="2019-03-11T12:33:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@c8b96ec]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
      </class> <!-- com.mendix.test.LoginScript -->
      <class name="com.mendix.test.MaterialScript">
        <test-method status="FAIL" signature="Material_Create_Syndication_Check(java.util.Map)[pri:0, instance:com.mendix.test.MaterialScript@4cf777e8]" name="Material_Create_Syndication_Check" duration-ms="1" started-at="2019-03-11T12:34:17Z" data-provider="Process_Information_Check" finished-at="2019-03-11T12:34:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{S.NO=9.0, OpCo=BE01, Test_Case=Create with ref Global Only - SAP Direct Copy with Rejections, Execute=Y, RequestId=1416843, Level1Global=GBDA, Level2Global=GDA, Level1Local=LDS, Level2Loca=LBDA, Syndication=Y, Global_ID=502871, Material_Number_AH1=, Material_Number_AE1=, Status_AH1=, Status_AE1=, Status_JDE=, Q=, R=, S=, T=, U=, V=, W=, X=, Y=, Z=, AA=, AB=, AC=, AD=, AE=, AF=, AG=, AH=, AI=, AJ=, AK=, AL=, AM=, AN=, AO=, AP=, AQ=, AR=, AS=, AT=, AU=, AV=, AW=, AX=, AY=, AZ=, BA=, BB=, BC=, BD=, BE=, BF=, BG=, BH=, BI=, BJ=, BK=, BL=, BM=, BN=, BO=, BP=, BQ=, BR=, BS=, BT=, BU=, BV=, BW=, BX=, BY=, BZ=, CA=, CB=, CC=, CD=, CE=, CF=, CG=, CH=, CI=, CJ=, CK=, CL=, CM=, CN=, CO=, CP=, CQ=, CR=, CS=, CT=, CU=, CV=, CW=, CX=, CY=, CZ=, DA=, DB=, DC=, DD=, DE=, DF=, DG=, DH=, DI=, DJ=, DK=, DL=, DM=, DN=, DO=, DP=, DQ=, DR=, DS=, DT=, DU=, DV=, DW=, DX=, DY=, DZ=, EA=, EB=, EC=, ED=, EE=, EF=, EG=, EH=, EI=, EJ=, EK=, EL=, EM=, EN=, EO=, EP=, EQ=, ER=, ES=, ET=, EU=, EV=, EW=, EX=, EY=, EZ=, FA=, FB=, FC=, FD=, FE=, FF=, FG=, FH=, FI=, FJ=, FK=, FL=, FM=, FN=, FO=, FP=, FQ=, FR=, FS=, FT=, FU=, FV=, FW=, FX=, FY=, FZ=, GA=, GB=, GC=, GD=, GE=, GF=, GG=, GH=, GI=, GJ=, GK=, GL=, GM=, GN=, GO=, GP=, GQ=, GR=, GS=, GT=, GU=, GV=, GW=, GX=, GY=, GZ=, HA=, HB=, HC=, HD=, HE=, HF=, HG=, HH=, HI=, HJ=, HK=, HL=, HM=, HN=, HO=, HP=, HQ=, HR=, HS=, HT=, HU=, HV=, HW=, HX=, HY=, HZ=, IA=, IB=, IC=, ID=, IE=, IF=, IG=, IH=, II=, IJ=, IK=, IL=, IM=, IN=, IO=, IP=, IQ=, IR=, IS=, IT=, IU=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mendix.test.MaterialScript.Material_Create_Syndication_Check(MaterialScript.java:512)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Material_Create_Syndication_Check -->
      </class> <!-- com.mendix.test.MaterialScript -->
    </test> <!-- Material_Create_Syndication_Check -->
  </suite> <!-- Create with ref Global Only-SAP Direct Copy with Rejections -->
</testng-results>
